### PR TITLE
Tune the max effort values for the simulation.

### PR DESCRIPTION
**Pull request by Roelemans**
Dependent on https://github.com/project-march/simulation/pull/28

Closes PM-101

## Description

Tune the max effort values for the simulation. This prevents safety controller from glitching, allowing replacement of endstops. As a result, the simulation and exo can use the same URDF again.

## Changes

* Discovered that the safety controller will cause problems without appropriate max effort values. 
* Max effort values of simulation tuned.
* Simulation URDF deleted.
* Simulation now hangs in the air by default.

### CODE REVIEW

File: march_description/urdf/march-iv.xacro [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
 <!-- Revolute-Revolute Manipulator -->
 <robot name="march" xmlns:xacro="http://www.ros.org/wiki/xacro">
+
     <!-- Switch between airgaiting or standing on the ground-->
-    <!--Value set to: 'world' fixes the robot to the world-->
-    <!--Value set to: 'base_link' makes the robot stand on the ground-->
-    <!--Important note: when changing value make sure rviz fixed frame is set to the same value (moveit.rviz)-->
+    <!-- Value set to: 'world' fixes the robot to the world-->
+    <!-- Value set to: 'base_link' makes the robot stand on the ground-->
+    <!-- Important note: when changing value make sure rviz fixed frame is set to the same value (moveit.rviz)-->
```
Comment (Olavhaasie) [inline march_description/urdf/march-iv.xacro]: (moveit.rviz)

Is this still relevant?

File: march_description/urdf/march-iv.xacro [hunk 2]
```diff
@@ -470,22 +481,33 @@
...
+        <mu2>${foot_mu2}</mu2>
+        <minDepth>${foot_min_depth}</minDepth>
+    </gazebo>
 
     <!-- Joints -->
     <joint name="hip" type="fixed">
         <parent link="${parent_name}"/>
         <child link="hip_base"/>
-        <origin xyz="0 0 ${lower_leg_height+upper_leg_height+hover_height}" rpy="0 0 0"/>
+        <origin xyz="0 0 ${lower_leg_height+upper_leg_height+ankle_plate_height+width/2.0+hover_height}" rpy="0 0 0"/>
```
Comment (Olavhaasie) [inline march_description/urdf/march-iv.xacro]: What does this fix?

Comment (Roelemans) [reply march_description/urdf/march-iv.xacro]: That ensures that when hover_height = 0 the exo is actually standing on the ground.

File: march_description/urdf/march-iv.xacro (hunk 1 above)
Comment (Roelemans) [reply march_description/urdf/march-iv.xacro]: Ah good spot, it is not.

**Pull request merged**